partition/androidbootenv/androidbootenv_test.go

```diff
@@ -0,0 +1,72 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2014-2017 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package androidbootenv_test
+
+import (
+	"path/filepath"
+	"testing"
+
+	. "gopkg.in/check.v1"
+
+	"github.com/snapcore/snapd/partition/androidbootenv"
+)
+
+// Hook up check.v1 into the "go test" runner
+func Test(t *testing.T) { TestingT(t) }
+
+type androidbootenvTestSuite struct {
+	envPath string
+}
+
+var _ = Suite(&androidbootenvTestSuite{})
+
+func (a *androidbootenvTestSuite) SetUpTest(c *C) {
+	a.envPath = filepath.Join(c.MkDir(), "androidbootenv")
+}
+
+func (a *androidbootenvTestSuite) TestSet(c *C) {
+	env := androidbootenv.NewEnv(a.envPath)
+	c.Check(env, NotNil)
+
+	env.Set("key", "value")
+	c.Check(env.Get("key"), Equals, "value")
+}
+
+func (a *androidbootenvTestSuite) TestSaveAndLoad(c *C) {
+	env := androidbootenv.NewEnv(a.envPath)
+	c.Check(env, NotNil)
```

Comment:
here this should probably be an `Assert` (otherwise all the other tests will panic)